Sources/MarshaledObject.swift

```diff
@@ -108,15 +108,15 @@ public extension MarshaledObject {
         }
     }
     
-    public func valueForKey<A: RawRepresentable where A.RawValue: ValueType>(_ key: KeyType) throws -> A {
+    public func valueForKey<A: RawRepresentable>(_ key: KeyType) throws -> A where A.RawValue: ValueType {
```

Comment:
There are a bunch of converted things like this. The where clause was moved to the end of the line.